src/explore-education-statistics-frontend/src/modules/find-statistics/__tests__/PublicationReleasePage.test.tsx

```diff
@@ -224,6 +235,18 @@ describe('PublicationReleasePage', () => {
     );
   });
 
+  test('renders national statistics image', () => {
+    const { container } = render(
+      <PublicationReleasePage release={testRelease} />,
+    );
+
+    expect(
+      container.querySelector(
+        'img[alt="UK statistics authority quality mark"]',
+      ),
+    ).toBeDefined();
```

Comment:
Shouldn't this be in `toBeTheDocument`? 

Additionally, `toBeDefined` checks if it's `undefined` or not, so this would most likely always pass as `querySelector` returns null if the element doesn't exist.

Reply:
Good spot! I just moved this test but didn't really look at what was in it. I've changed it and the test for the image not showing for official stats now.